update-explorer-source.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+if [ "X$EXPLORER_STRINGS_SHEET" = "X" ]; then
+  echo "Set environment variable EXPLORER_STRINGS_SHEET to the URL of the published CSV"
```

Comment:
I think we should hardcode it into the script so that it's easier to use. The data in the spreadsheet is public anyways